functions/EmuScripts/emuDeckRetroArch.sh

```diff
@@ -1541,6 +1543,35 @@ RetroArch_melonDS_setConfig(){
 	RetroArch_setOverride 'nds.cfg' 'melonDS'  'rewind_enable' '"false"'
 }
 
+RetroArch_melonDSDS_setUpCoreOpt(){
+	RetroArch_setOverride 'melonDS DS.opt' 'melonDS DS'  'melonds_audio_bitrate' '"Automatic"'
+	RetroArch_setOverride 'melonDS DS.opt' 'melonDS DS'  'melonds_audio_interpolation' '"None"'
```

Comment:
Equivalent to `melonds_audio_interpolation = "disabled"`, which is the default behavior.